spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+# You can directly load the secret into a variable like this example
+# This demonstrates multiple ways of loading the same application secret using template syntax.
+#
+# Please refer to the documentation for the protocol syntax.
+#
+# Examples:
+# my-app-secret-3=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret}
+# my-app-secret-4=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret/version/latest}
+# my-app-secret-5=${sm://<YOUR_PROJECT_ID>/application-secret/latest}
```

Comment:
It's not clear whether someone should use the `projects/` prefix or not - is there some sort of behavioral difference? If there is no functional difference, I'd simplify by only including one form (probably without the prefix).

Reply:
Done.

To simplify the explanation, I now just share that you can specify secrets via `${sm://SECRET_ID/VERSION}`. Then I added a note -- "If you would like to specify secrets from a different project, you can use `${sm://PROJECT/SECRET_ID/VERSION}`."